packages/schemas/src/text/constants.ts

```diff
@@ -21,3 +22,83 @@ export const DEFAULT_DYNAMIC_MIN_FONT_SIZE = 4;
 
 export const DEFAULT_DYNAMIC_MAX_FONT_SIZE = 72;
 export const FONT_SIZE_ADJUSTMENT = 0.25;
+
+export const LINE_START_FORBIDDEN_CHARS = [
```

Comment:
I'm looking for a better way to manage such rules.  
This issue occurs not only in Japanese but also in various languages such as Chinese, Korean, and Thai.  
If there is a dataset that manages these characters, I would like to use it.

Reply:
Probably we need a way to handle language-specific rules, but that might reintroduce `locale`...

Some characters are common, but others are not.

The following blog posts by Morisawa are very informative. We may be able to define four rules as in Adobe InDesign, then create a set of rules for each language.

https://note.morisawa.co.jp/n/n64589ad7572d

However, this can be quite challenging without support from native speakers.